src/cephadm/tests/test_cephadm.py

```diff
@@ -1433,11 +1433,157 @@ def test_distribute_ssh_keys(self, call):
         assert retval == 1
 
 
+class TestSNMPGateway:
+    V2c_config = {
+        'snmp_community': 'public',
+        'destination': '192.168.1.10:162',
```

Comment:
ipv4 is easy. does it also work for ipv6?

Reply:
I have an issue open on the snmp_notifer project about just that. Using an ipv6 address as a target (outside of the ceph cluster) doesn't appear to work.

Reply:
I have an issue open on the snmp_notifer project about just that. Using an ipv6 address as a target (outside of the ceph cluster) doesn't appear to work.

I'm also adding a valid_addr function so we can check for name, ipv4 or ipv6 type and will include that in the spec.validate method to remove ipv6 a a destination at least for now.

Reply:
> Using an ipv6 address as a target doesn't appear to work.

Sigh. Now what? Just don't care about ipv6 or just use generic functions assuming it will work at some time in the future?